CMakeLists.txt

```diff
@@ -61,7 +61,21 @@ endif()
 # Find OGRE
 list(APPEND ign_ogre_components "RTShaderSystem" "Terrain" "Overlay")
 
-ign_find_package(IgnOGRE VERSION 1.8.0
+# Ogre versions greater than 1.9 are not officialy supported.
+# Display a warning for the users on this setup unless they provide
+# USE_UNOFFICAL_OGRE_VERSIONS flag
+if (NOT USE_UNOFFICAL_OGRE_VERSIONS)
+  ign_find_package(IgnOGRE VERSION 1.10
+    COMPONENTS ${ign_ogre_components})
+
+  if (OGRE_FOUND)
+    IGN_BUILD_WARNING("Ogre 1.x versions greater than 1.9 are not officially supported."
+                      "The software might compile and event work but support from upstream"
+                      "could be reduced to accept patches for newer versions")
+  endif()
+endif()
+
+ign_find_package(IgnOGRE VERSION 1.9.0
```

Comment:
I think we can skip looking for 1.9.0 if 1.10 is found?

Reply:
how about this 90ebd1e ?